docs/support/logging/Ch-Logging.rst

```diff
@@ -147,62 +112,75 @@ Configuration Properties
 +---------------------------------------------------------+-------------------------------------+---------------------------------------------------------------------------+
 
 
-| \*means the configuration value can be changed if necessary.
-| \**means the configuration value has to be replaced.
+| \*means the configuration value can be changed on the fly if using a configuration registry (like Consul).
+| \**means the configuration value can be changed but the service must be restarted.
 | \***means the configuration value should NOT be changed.
 
 
 ====================================================
-Logging Service Client Library for Java (Deprecated)
+Logging Service Client Library for Go
 ====================================================
 
-As most of EdgeX Foundry microservices are implemented in Java, we provide a Client Library for Java, so that Java-based microservices could directly switch their Loggers to use EdgeX Foundry Logging Service.  The next graphic shows the high-level design architecture for the Java Client Library.
+As the reference implementation of EdgeX Foundry microservices is written in Go, we provide a Client Library for Go so that Go-based microservices could directly switch their Loggers to use the EdgeX Foundry Logging Service.
+
+The Go LoggingClient is part of the .. _go-mod-core-contracts module: https://github.com/edgexfoundry/go-mod-core-contracts. This module can be imported into your project by including a reference in your go.mod. You can either do this manually or by executing "go get github.com/edgexfoundry/go-mod-core-contracts" from your project directory will add a reference to the latest tagged version of the module.
```

Comment:
Link format needs to be updated as described in comment above

Reply:
Change was made